easybuild/easyconfigs/p/pgi/pgi-15.7-GNU-4.9.2-2.25.eb

```diff
@@ -0,0 +1,22 @@
+name = 'pgi'
```

Comment:
My reasoning was as follows (see the comment thread for the easyblock)

The compiler itself ships with a sample modulefile named "pgi/15.7", so people are used to loading modulefiles for this compiler with the non-capitalized pgi (you can do a google search for "pgi module" and see what other sites use). 


On the other hand this also seems to go with an all-lowercase convention that many HPC sites but evidently not easybuild use.